types/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+    "compilerOptions": {
+      "lib": ["es2015"],
+      "strict": true,
+      "baseUrl": ".",
+      "paths": {
+        "remark-unwrap-images": ["."]
+      }
+    }
+  }
```

Comment:
could prettier be run on this file, to align indentation and add the missing line feed at the end of file?